src/core/Expander/Expander/__snapshots__/Expander.test.tsx.snap

```diff
@@ -112,28 +112,37 @@ exports[`Basic expander shoud match snapshot 1`] = `
   box-sizing: border-box;
 }
 
-.c5 {
-  vertical-align: baseline;
-}
-
-.c5.fi-icon {
-  display: inline-block;
-}
-
-.c5 .fi-icon-base-fill {
-  fill: currentColor;
-}
-
-.c5 .fi-icon-base-stroke {
-  stroke: currentColor;
```

Comment:
Why have all these snapshots changed? 🤔

Reply:
I don't know. This has happened at least once before with some random PR.
I guess once in a while something mysterious happens with the system that creates the snapshots and css classes are spit out in a different order.  

Googling hints that this might have something to do with styled components
https://github.com/styled-components/jest-styled-components/issues/289